jib-plugins-extension-common/src/main/java/com/google/cloud/tools/jib/plugins/extension/JibPluginExtensionException.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.plugins.extension;
+
+import java.util.concurrent.ExecutionException;
+
+/** Exception while running Jib plugin extensions. */
+public class JibPluginExtensionException extends ExecutionException {
```

Comment:
It doesn't feel like `ExecutionException` is the right Exception to extend? Do we even need to really extend an exception like this?

Reply:
Ah, good catch. Leftover from previous POC iterations.